src/azure-cli-core/azure/cli/core/auth/landing_pages/success.html

```diff
@@ -2,11 +2,29 @@
 <html>
 <head>
     <meta charset="utf-8" />
-    <meta http-equiv="refresh" content="10;url=https://docs.microsoft.com/cli/azure/">
+    <meta http-equiv="refresh" content="120;url=https://docs.microsoft.com/cli/azure/">
     <title>Login successfully</title>
+    <style>
+        body {
+            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; 
+        }
+
+        code {
+            font-family: 'Courier New', Courier, monospace;
+            display: inline-block;
+            font-weight: bold;
+            background-color: rgb(229, 229, 229);
+            padding: 12px 16px;
+            margin: 8px 0px;
+        }
+    </style>
 </head>
 <body>
-    <h4>You have logged into Microsoft Azure!</h4>
-    <p>You can close this window, or we will redirect you to the <a href="https://docs.microsoft.com/cli/azure/">Azure CLI documentation</a> in 10 seconds.</p>
+    <h3>You have logged into Microsoft Azure!</h3>
+    <p>You can close this window, or we will redirect you to the <a href="https://docs.microsoft.com/cli/azure/">Azure CLI documentation</a> in 2 minutes.</p>
+    <h3>Announcements</h3>
+    <p>Starting in May 2023, Azure CLI will authenticate on Windows using the <a href="https://learn.microsoft.com/en-us/windows/uwp/security/web-account-manager">Web Account Manager</a> (WAM) broker by default.</p> 
```

Comment:
I think it is better to make it clear that only Windows is supported, so that we won't bother Linux or MacOS users.

```suggestion
    <p>[Windows only] Starting in May 2023, Azure CLI will authenticate using the <a href="https://learn.microsoft.com/en-us/windows/uwp/security/web-account-manager">Web Account Manager</a> (WAM) broker by default.</p>
```